panos/network.py

```diff
@@ -1742,10 +1742,14 @@ class StaticRoute(VersionedPanObject):
         interface (str): Next hop interface
         admin_dist (str): Administrative distance
         metric (int): Metric (Default: 10)
-
+        enable_path_monitor (bool): Enable Path Monitor
+        failure_condition: Path Monitor failure condition set 'any' or 'all' 
```

Comment:
Just missing the `(str)` type in this docstring.

Reply:
@shinmog thanks a lots for your comment and tips, i will check and make the appropriate change 👍 have a good night :)